app/models/user.rb

```diff
@@ -1,2 +1,11 @@
 class User < ApplicationRecord
+  has_many :posts
+  has_many :comments
+  has_many :likes
+  validates :name, presence: true, length: { minimum: 3, maximum: 20 }
+  validates :posts_counter, numericality: { greater_than_or_equal_to: 0 }
+
+  def recent_posts(limit = 3)
+    posts.order(created_at: :desc).limit(limit)
+  end
```

Comment:
- [ ] Great job so far @Reggeditt  There are just a few changes I will suggest you implement and you'll be on your way to the next milestone :+1: 
- [ ] Per the project requirements, you were asked to `Add unit specs for all of your models' methods and validations` You have done an amazing job adding most of the tests. I noticed that this `recent_posts` method hasn't been tested yet. So kindly include that in your tests :wink: